recipe/meta.yaml

```diff
@@ -22,22 +22,24 @@ requirements:
     - setuptools >=61
     - pip
   run:
-    - python >={{ python_min }}
+    - python >=3.10
     - einops >=0.3
-    - lightning >=2.1.1
-    - matplotlib-base >=3.3.3
-    - numpy >=1.19.3
+    - lightning >=2.4
+    - matplotlib-base >=3.5
+    - numpy >=1.21.1
     - pandas >=1.1.3
     - pytorch >=2.0
     - torchmetrics >=1.2
     - torchvision >=0.16.1
     - scikit-learn >=1.3
     - gpytorch >=1.11
-    - laplace-torch >=0.1a2
+    - laplace-torch >=0.2.1
     - uncertainty-toolbox >=0.1.1
     - kornia >=0.6.9
     - timm >=0.9.2
     - torchseg >=0.0.1a1
+    - h5py >=3.12.1
+    - ema-pytorch >=0.7
```

Comment:
Let's try `ema_pytorch` and see if that works, not sure if conda is underscore-sensitive (pip is not)